.github/workflows/crowdin-contributors.yml

```
@@ -44,6 +44,7 @@ jobs:
       - name: Create Pull Request
         uses: peter-evans/create-pull-request@v7
         with:
+          token: ${{ secrets.CROWDIN_ACTION_TOKEN }}
```

```suggestion
          token: ${{ secrets.GITHUB_TOKEN }}
```

[This block](https://github.com/the-turing-way/the-turing-way/pull/4323/files#diff-264b5bf640da600c6608ec79c90839d8fe4fd6d7f39c916517e8a10a8539defbL14-L16) sets the proper permissions on `GITHUB_TOKEN` (for this workflow only), but it wasn't being passed to the action. If we do it this way, we won't have to manage a secret in the repo.